### ISSUE TITLE
IOS XE: "show ethernet service instance detail" not parsing description with space

### ISSUE THREAD

**Issue by lzuzcak** [CiscoTestAutomation/genieparser#84]
Hello team

the parser for command "show ethernet service instance detail" will not parse description in case it have space in it.

#no spaces, working fine
```
description = "Description: example_string"
p3 = re.compile(r'^Description: +(?P<description>\S+)$')
output -> {'description': 'example_string'}
```

#space in description, not working
```
description = "Description: example string"
p3 = re.compile(r'^Description: +(?P<description>\S+)$')
output -> empty
```

I have checked regexp from "show interfaces description" and that one works fine.

#no spaces, working fine
```
description = "Description: example_string"
p3 = re.compile(r'^Description: +(?P<description>(.*))?$')
output -> {'description': 'example_string'}
```

#space in description, working fine
```
description = "Description: example string"
p3 = re.compile(r'^Description: +(?P<description>(.*))?$')
output -> {'description': 'example string'}
```

I would appreciate if this could be fixed in new versions.

Thank you

LZ

******************
Attaching my testing code in case that you would like to play with it

```
 description = "Description: example string"
    p3 = re.compile(r'^Description: +(?P<description>(.*))?$')
    #p3 = re.compile(r'^Description: +(?P<description>\S+)$')
    match = p3.match(description)
    if match:
        group = match.groupdict()
        print(group)
```

**Comment by xiaoxinz-cisco** [comment]
Thank you for reporting. We will look into the issue and get back to you.

**Comment by xiaoxinz-cisco** [comment]
Hello @lzuzcak 

We fixed the issue and genie package **v20.1.1b0** has been released with the fix.
Could you please run the following command and try again to see if it works?

**pip install genie.libs.parser --upgrade --pre**

Please try and let's know if there is an issue.

Thanks,
Irene

**Comment by lzuzcak** [comment]
Hello Irene

Thank you, fix is working fine.

LZ
